src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -94,14 +94,27 @@ export class AddressForm extends React.Component<
 
   render() {
     const lockCountryToOrigin = !this.props.billing && this.props.domesticOnly
+
+    // FIXME: There's an existing iOS bug which uppercases the first two letters when autoCapitalization="word" is enabled
+    // so we are for now disabling autoCapitalization on mobile
+    // https://bugs.webkit.org/show_bug.cgi?id=157410
+    let autoCapitalizeWord = null
+    if (
+      typeof window !== "undefined" &&
+      window.innerWidth &&
+      window.innerWidth > 768
+    ) {
+      autoCapitalizeWord = "words"
+    }
```

Comment:
This makes sense, though I'm not sure if we need something so complex. I think removing the `autoCapitalize="words"` altogether might be an acceptable solution, since `autoCapitalize` isn't required for desktop browsers (and iPad would likely have the same bug as iPhone). I think removing `autoCapitalize="words"` in all cases would be a better approach, but I'm open to feedback on this 👍

Reply:
I agree w/ this.